---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* PhD Student in Biomedical Enginnering, University of Toronto, ON, CA, 2024 - Present
* HBSc. in Bioinformatics and Computational Biology, University of Toronto, 2020 - 2024

Research experience
======
* Deciphering 3’ UTR mediated gene regulation using interpretable deep representation learning
  * Zhang Laboratory, University of Toronto, Terrence Donnelly Centre for Cellular and Biomolecular Research (CCBR)
  * 2022/04 - 2023/08, Advisor: Dr.Zhaolei Zhang
  * Processed the sequence data of human mature mRNA at transcriptome level
  * Built a prediction model that can predict mRNA sub-cellular localization, binding site of RBPs and m6A modification sites in mRNA sequences based on a BERT model with other members of the Laboratory
  * Compared performances of different state-of-the-art computing models on the same dataset
  * Designed and implemented m6A analytical experiment, and extracted Motif features of mRNA sequences
  * Conducted analysis of sequence point mutation and visual analysis of model attention mechanism
  * Visualized data, model structure, and performances with multiple techniques

* Making use of Protein-protein interaction network and machine learning techniques to identify potentially new cancer driver genes
  * Reimand Laboratory, University of Toronto, Ontario Institute for Cancer Research (OICR) 
  * 2023/04 - 2023/08, Advisor: Dr.Jüri Reimand
  * Constructed a graph based on the Protein-protein interaction network, and generated embedding features of each node by Node2Vec
  * Ran multiple diagnostic analyses on the generated features and model outputs
  * Trained and tested logistic regression classifier to make classification on different hallmark gene sets
  * Tried to find the false positive genes that are misclassified multiple times, which might be potential cancer driver genes Github link: https://github.com/reimandlab/PPI_network_based_gene_type_prediction_model

* Developing computational pipelines for analysis of spatially resolved sequence data based on Slide-seq technique to investigate the mechanism of deficiency of nervous system of Downsyndrome patients
  * Kalish Laboratory, University of Toronto, The Hospital for Sick Children (SickKids)
  * 2023/09 - 2024/08, Advisor: Dr.Brian Kalish
  * Aligned sequencing data using STAR
  * Deduplicated reads using unique molecular identifiers (UMI) tools Analysis and visualization by Seurat
  * Integrated scRNA-seq data, as a reference, with Slide-seq data for labelling cell types with machine learning technique
  * Developed a lasso selection tool specifically for Slide-seq data

Honours and Scholarships
======
* Dean’s List Scholar 06/2022   
* The Jean (MacIntosh) MacLeod Scholarship 08/2022
* Dean’s List Scholar 06/2023
* The Anne Oaks Scholarship 07/2023
